modules/claims_api/spec/requests/v2/veterans/intent_to_files_request_spec.rb

```diff
@@ -411,7 +411,7 @@
                 with_okta_user(scopes) do |auth_header|
                   survivor_data = data
                   survivor_data[:type] = 'survivor'
-                  survivor_data[:claimantSsn] = '123456'
+                  survivor_data[:claimantSsn] = '123456789'
```

Comment:
This isn't testing that the `gsub` is working to remove the dashes & parse the SSN as valid.`123-45-6789` would ensure this passes as expected :)

Reply:
Added a test for claimantSsn with separators in [PR](https://github.com/department-of-veterans-affairs/vets-api/pull/11975) for API-24720.